frontend/public/components/machine.tsx

```diff
@@ -94,7 +94,7 @@ const MachineTableHeader = () => {
 };
 MachineTableHeader.displayName = 'MachineTableHeader';
 
-const getMachinePhase = (obj: MachineKind) => {
+export const getMachinePhase = (obj: MachineKind) => {
```

Comment:
We should move this to a separate module, otherwise the machine pages will be bundled with all tables. Probably not a huge difference in this case, but it's a better pattern.